components/Cta/CtaContent.tsx

```diff
@@ -31,12 +31,11 @@ export const CtaContent = ({
   return (
     <FlexBox as="span" alignItems="center">
       {iconPosition === 'right' && children}
+      {/* Use this whitespace-nowrap trick so icon won't get pushed to the next line on its own */}
       {heroicon && (
-        <HeroIcon
-          icon={heroicon}
-          className={cnb(styles.icon, iconStyle, iconAnimate, iconMarginLeft, iconMarginRight, iconClasses)}
-          {...iProps}
-        />
+        <span className="whitespace-nowrap">
+          &#65279;<HeroIcon icon={heroicon} className={cnb(styles.icon, iconStyle, iconAnimate, iconMarginLeft, iconMarginRight, iconClasses)} {...iProps} />
+        </span>
```

Comment:
I used this trick to prevent icon from wrapping to 2nd line on its own. Used it on Directory and works well.